files/default/neutron-ha-tool.py

```diff
@@ -305,14 +304,15 @@ def run(args):
     return 1 if errors > 0 else 0
 
 
-def l3_agent_rebalance(qclient, noop=False, wait_for_router=True):
+def l3_agent_rebalance(qclient, args):
```

Comment:
I think that this change can break stuff. You are replace some optional parameters that now needs to be inside args and are not optional anymore. Did you checked this?

Also replacing from 3 to 2 arguments is not a big gain, and you hide the semantic of the signature. Also for the rest